index.bs

```diff
@@ -9725,6 +9725,31 @@ corresponding to [=interface members=].
 </div>
 
 
+<h4 id="Historical" extended-attribute lt="Historical">[Historical]</h4>
+
+When the [{{Historical}}] [=extended attribute=] appears on an individual
+[=dictionary member=], [=interface member=], [=interface mixin member=], or
+[=namespace member=], it indicates that the construct exists for historical
+reason.
+
+The [{{Historical}}] extended attribute must
+[=takes no arguments|take no arguments=].
+
+<div class="example">
+    There are Web APIs that have been replaced by better new APIs and are
+    generally discouraged to use. Code editors can consider this extended
+    attribute as a hint to warn users where such APIs are used. Browser
+    implementations may also consider this as a suggestion to add use counters.
```

Comment:
I actually don't think it's worth the effort of removing these APIs. They are not harmful and removing them might break the end user experience on sites, causing frustration for most parties involved.

Reply:
I added this as I thought Gecko already had use counters for them but it seems not. I'm okay without this.